packages/flutter/lib/src/material/dialog.dart

```diff
@@ -931,15 +931,15 @@ Widget _buildMaterialDialogTransitions(BuildContext context, Animation<double> a
 /// barrier will dismiss the dialog. It is `true` by default and can not be `null`.
 ///
 /// The `barrierColor` argument is used to specify the color of the modal
-/// barrier that darkens everything the dialog. If `null` the default color
+/// barrier that darkens everything below the dialog. If `null` the default color
```

Comment:
nit: maybe `behind` instead of `below`
```suggestion
/// barrier that darkens everything behind the dialog. If `null` the default color
```

Reply:
Yes I wanted to do behind, but above/below is used everywhere when describing such behavior, even on the ModalBarrier page. Maybe they were describing the widget tree?

Reply:
Let's just keep "below" then. I did a quick search around the codebase and below and behind are both used in different contexts, so it should be pretty clear what either means to our users